### ISSUE TITLE
bug in step1-03,Extra brackets

### ISSUE THREAD

**Issue by yinsang** [Microsoft/frontend-bootcamp#96]
https://github.com/Microsoft/frontend-bootcamp/blob/master/step1-03/demo/README.md
```js
const button = document.querySelector('.submit');
button.onclick = displayMatches();
```
It's wrong to add brackets after displayMatches.

**Comment by ahtee** [comment]
Can you go into further detail on why? Syntactically it makes sense that displayMatches is a function when including parentheses() and variables or constants can be quickly seen as such because they wont have parentheses()

**Comment by ecraig12345** [comment]
@yinsang Good catch, thanks!

@ahtee JavaScript has first-class functions, meaning that functions can be assigned to variables or passed as parameters the same as any other object or value. In this case, what we want to do is set the `displayMatches` function itself (*not* the return value from calling it) to be the button's `onclick` handler. The button will call `displayMatches` later, when it's clicked.

I don't think the step text should go into all that detail (unless it turns out to be a common point of confusion), but I updated the wording a bit to hopefully clarify what's going on.

> In our example, we want to trigger a function when a button is clicked. To do this, we first need to get a reference to the button. We can use the `querySelector` method of the browser-provided `document` global variable to get that reference. **Then we can set our `displayMatches` function to be the button's `onclick` handler.**

**Comment by micahgodbolt** [comment]
fixed in #98
